libobs/obs-data.c

```diff
@@ -27,11 +27,20 @@
 #include "obs-data.h"
 
 #include <jansson.h>
+#include <uthash/uthash.h>
+
+#ifdef UTHASH_USE_OBS_ALLOCATOR
+#undef uthash_malloc
+#undef uthash_free
+#define uthash_malloc(sz) bmalloc(sz)
+#define uthash_free(ptr, sz) bfree(ptr);
+#endif
```

Comment:
This is duplicated across many source files, since we're vendoring uthash could we move it to `uthash.h`? Also doesn't this need to be set before the header is included?

Reply:
It does not need to be set before the header is included, this works as expected. Since we want to eventually move away from vendored deps and want to avoid modifications I'd probably rather just remove the commit for this entirely, it was mostly just to sanity check what I'm doing anyway.